tests/local-test.py

```diff
@@ -0,0 +1,32 @@
+import json
+from selenium import webdriver
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.options import Options as ChromeOptions
+
+options = ChromeOptions()
+driver = webdriver.Remote(
+    command_executor='http://localhost:4444/wd/hub',
```

Comment:
Don't think sample should ideally point to local port.

Reply:
@alliv8 thoughts?

Reply:
Yea it's good to show how our demo works and that the SDK works with any web driver

Reply:
Let's add a code comment that this will be overridden when invoking the sdk